scripts/azure-linux_mac-release.yml

```diff
@@ -18,15 +18,9 @@ steps:
     # This stage is needed until https://github.com/actions/runner-images/pull/7125 or similar is merged
     # MacOS image 20230214.1 removed implicit installation of pkg-config
     set -e pipefail
-    brew install pkg-config
+    brew install pkg-config autoconf automake
   condition: eq(variables['Agent.OS'], 'Darwin')
-  displayName: 'Install brew package for pkg-config (OSX only)'
-
-- bash: |
-    set -e pipefail
-    brew uninstall --ignore-dependencies libidn2 brotli rtmpdump
```

Comment:
We had removed these packages originally to prevent curl from linking against them and causing undefined lib errors at runtime. Are we sure its safe to remove the uninstalls?

Reply:
Yes we've already added exclusions to the configure call, including these: https://github.com/TileDB-Inc/TileDB/blob/dev/cmake/Modules/FindCurl_EP.cmake#L168-L192

Reply:
Okay sounds good.

-------

I do reserve the right to say "I told you so" if any problems arise :rofl: